src/main/resources/graphql/service/quiz.graphqls

```diff
@@ -134,10 +134,226 @@ type MultipleChoiceAnswer {
     feedback: String
 }
 
+type ClozeQuestion implements Question {
+    """
+    The elements of the cloze question.
+    """
+    clozeElements: [ClozeElement!]!
+    """
+    All selectable answers for the blanks (computed). This contains the correct answers as well as wrong answers.
+    """
+    allBlanks: [String!]!
+    """
+    Whether the blanks must be answered in free text or by selecting the correct answer from a list.
+    """
+    showBlanksList: Boolean!
```

Comment:
Don't know if necessary. There could be two modes: one where you have to select the gaps from a list, and one where you have to enter the text.

Reply:
with list do you mean a multiple choice list for each blank or a list that contains all the answers of the whole text so you have to "sort" them into the different gaps?

Reply:
not a multiple choice for each blank, but a common pool of answers where you drag it to the right place

Reply:
So you mean a list with all possible answers. Is there a possibility that there are more answer possibilites than blanks? And is it possibile to have exactly the number of answers possibilites as you have blanks?

Reply:
Yes, there will a "wrongAnswers" field, containing additional possibilities